tests/unit/parsers/test.manifestjson.js

```diff
@@ -2248,6 +2248,24 @@ describe('ManifestJSONParser', () => {
         expect(warningsV3.length).toEqual(6);
       }
     );
+
+    // See: https://github.com/mozilla/addons-linter/issues/5194
+    it('should handle non-string values', () => {
+      const addonLinter = new Linter({ _: ['bar'] });
+      const json = validManifestJSON({ content_security_policy: [true] });
```

Comment:
@willdurand would you mind to also add a test manifest with a non string csp value when content_security_policy is in the object form? (e.g. even just making this same test executed for both this value and `{ content_security_policy: { extension_pages: true } }`).